security/run.sh

```diff
@@ -1,16 +1,19 @@
 #!/bin/bash
 PK="training:security"
+HOST=http://localhost:9000
+
 
 if [ "$SQ_URL" != "" ]; then
-   sqHostOpt="-Dsonar.host.url=$SQ_URL"
+   HOST=$SQ_URL
 fi
+
+sqHostOpt="-Dsonar.host.url=$SQ_URL"
+
 if [ "$TOKEN" != "" ]; then
    sqLoginOpt="-Dsonar.login=$TOKEN"
+   echo "Deleting project"
+   curl -X POST $curlLoginOpt $HOST/api/projects/delete?project=$PK
```

Comment:
Seems that $curlLoginOpt is  a new unset parameter. Coming from the environment ?